packages/terra-status-view/src/StatusView.jsx

```diff
@@ -36,223 +43,148 @@ const propTypes = {
   isGlyphHidden: PropTypes.bool,
 
   /**
-   * The descriptive text to display.
+   * The descriptive text, displayed under the title.
    */
   message: PropTypes.string,
 
   /**
-   * The status view's title to display. Status views with variants of type `no-data`,
-   * `no-matching-results`, `not-authorized`,
-   * `error` will have defaulted values unless overridden with this prop.
+   * The title displayed under the glyph. Variants contain default titles that can be overriden by this prop.
    */
   title: PropTypes.string,
 
   /**
-   * Determines the glyph to display, one of the following: `no-data`,
-   * `no-matching-results`, `not-authorized`,
-   * `error`, or `custom`.
+   * Sets the glyph and title using a pre-baked variant. One of the following: `no-data`,
+   * `no-matching-results`, `not-authorized`, or `error`
    */
   variant: PropTypes.oneOf(Object.values(StatusViewVariants)),
```

Comment:
Should this be required or can you implement a custom status view by not providing a variant?

Reply:
talked offline - `title`, `message`, and/or `customGlyph` props will override the defaults. I thought it was redundant to set the variant to `custom` in order to render those first class overrides. I'll talk to @neilpfeiffer on what use cases exist for custom status views. Maybe we can remove that functionality

Reply:
Custom is rare, so ideally most uses should/will choose a prop as their default choice. I think this is a cleaner API to not forcibly need to reset to custom (which is really just an 'un-set' variant). We can update documentation to more prominently encourage variant picking or discouraging custom-use if we need to.